rskj-core/src/test/java/org/ethereum/rpc/Web3ImplTest.java

```diff
@@ -93,12 +93,12 @@
  */
 public class Web3ImplTest {
 
-    public static final String BALANCE_10K = "0x" + ByteUtil.toHexString(BigInteger.valueOf(10000).toByteArray());
+    private static final String BALANCE_10K_HEX = "0x2710"; //10.000
     private final TestSystemProperties config = new TestSystemProperties();
```

Comment:
I've replaced directly with the constant. Let me know if you want to prefer to add a method like
`   private static String convertToHexString(int value) {
        return "0x" + ByteUtil.toHexString(BigInteger.valueOf(value).toByteArray());
    }
`

Reply:
nice!